nyc_data/ppe/views.py

```diff
@@ -79,12 +79,24 @@ def post(self, request):
                 return render(request, "upload.html",
                               UploadContext(error="Import already in progress for this file type",
                                             import_in_progress=ex.import_id)._asdict())
+            
+            except data_import.NoMappingForFileError as ex:
+                return render(request, "upload.html",
+                              UploadContext(error="No mapping found for this file")._asdict())
+            
 
 
 class Verify(View):
     def get(self, request, import_id):
         import_obj = DataImport.objects.get(id=import_id)
-        return render(request, "verify_upload.html", dict(import_id=import_id, delta=import_obj.compute_delta()))
+        #table = aggregations.AggregationTable(list(aggregation.values()))
```

Comment:
putting this in as a comment to build on top of, so we can insert a table of new rows instead of an unstyled bulleted point list